Content.Server/NPC/HTN/HTNSystem.cs

```diff
@@ -133,6 +133,31 @@ private void OnHTNShutdown(EntityUid uid, HTNComponent component, ComponentShutd
         component.PlanningJob = null;
     }
 
+    public void SetHTNEnabled(Entity<HTNComponent> ent, bool state, float planCooldown = 0f)
+    {
+        if (ent.Comp.Enabled = state)
+            return;
+
+        ent.Comp.Enabled = state;
```

Comment:
The if condition assigns the `state` variable into the enabled field. I assume you want this method to avoid running it's logic if the HTN is already enabled/disabled.
```suggestion
        if (ent.Comp.Enabled == state)
            return;

        ent.Comp.Enabled = state;
```